cmake/InstallLayout.cmake

```diff
@@ -220,12 +220,12 @@ set(CPACK_RPM_SERVER-VERSIONED_PACKAGE_REQUIRES
 set(CPACK_COMPONENT_SERVER-EL7_DISPLAY_NAME "foundationdb-server")
 set(CPACK_COMPONENT_SERVER-DEB_DISPLAY_NAME "foundationdb-server")
 set(CPACK_COMPONENT_SERVER-TGZ_DISPLAY_NAME "foundationdb-server")
-set(CPACK_COMPONENT_SERVER-VERSIONED_DISPLAY_NAME "foundationdb-server-${PROJECT_VERSION}")
+set(CPACK_COMPONENT_SERVER-VERSIONED_DISPLAY_NAME "foundationdb-${PROJECT_VERSION}-server")
```

Comment:
```suggestion
set(CPACK_COMPONENT_SERVER-VERSIONED_DISPLAY_NAME "foundationdb-server${PROJECT_VERSION}")
```

First tests with the `foundationdb-<component><version>` naming were conclusive. I suggest we try this name, as it follows [the official packaging guidelines for Fedora](https://docs.fedoraproject.org/en-US/packaging-guidelines/Naming/#multiple), useful for all RPM-based distributions.

Reply:
This example indicates `foundationdb6.3.20-server` and `foundationdb6.3.20-clients` is more consistent with the guidelines:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/65370928/133642774-17206067-2a9e-476d-a23f-915798859ecd.png">